src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FinalTransferConfirmationFields.scala

```diff
@@ -31,11 +29,6 @@ object FinalTransferConfirmationFields {
 
   implicit val addFinalTransferConfirmationType: ObjectType[Unit, FinalJudgmentTransferConfirmation] =
```

Comment:
You can remove this as it was only used in the `addFinalJudgmentTransferConfirmation` which you have now removed on line 50